src/main/java/core/basesyntax/FileWriterImpl.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class FileWriterImpl implements FileWriter {
+    @Override
+    public void write(String data, String filePath) throws IOException {
```

Comment:
The method `write` should handle exceptions more gracefully. Instead of throwing `IOException`, catch it and rethrow a `RuntimeException` with an informative message, as per the checklist.